src/renderer/stores/Workspace.ts

```diff
@@ -59,6 +61,10 @@ export class Workspace {
       this.showSidebar = !this.showSidebar;
       return false;
     });
+    //Listen for event from main process. 
+    ipcRenderer.on('openProject', () => {
+      this.openProject();
```

Comment:
We better do a `this.closeProject()` before `this.openProject()` so the previous runner would be terminated and the files would be cleared.